code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -77,6 +77,15 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/engineer/syndie
+
+/obj/item/weapon/storage/box/engineer/syndie/New()
+	..()
+	contents = list()

```

Comment:
why not /obj/item/weapon/storage/box/syndie? so you don't have to do that hacky stuff.